app/styles/components/shared/pricing.scss

```diff
@@ -2,54 +2,83 @@
 
 .pricing {
 	display: flex;
-	margin-top:20px;
+	margin: 20px 0;
+	user-select: auto !important;
 
 	&-switch {
 		display: flex;
 		justify-content:center;
 
 		&-item {
-			margin-right: 16px;
 			font-size: 17px;
 			padding: 10px 20px;
 			border-radius: 5px;
 			font-weight: 400;
-			cursor: pointer;
 			background: $white;
 			color: $green;
 			border: 1px solid $green;
 			transition: background 0.3s ease, color 0.3s ease;
 
-			&:last-of-type {
-				margin-right: 8px;
+			&:first-child {
+				border-radius: 5px 0 0 5px;
+			}
+
+			&:last-child {
+				border-radius: 0 5px 5px 0;
+			}
+
+			&:hover {
+				background-color: lighten($green, 20%);
+				color: $white;
 			}
 
-			&.is-active, &:hover {
+			&.is-active {
 				background-color: $green;
 				color: $white;
 			}
 		}
 	}
 
 	&-item {
+		cursor: pointer;
 		position: relative;
 		border-radius: 5px;
 		margin: 0 8px;
-		height:100%;
-		cursor:pointer;
 		border: 1px solid $light-grey;
-		background-color: $lightest-grey;
 		transition: transform 0.3s ease;
 		width: calc(50% - 16px);
 
 		&:hover {
 			transform: scale(1.02);
 		}
 
+		&:focus {
+			outline: none;
+		}
+
 		&:last-of-type {
 			margin-right: 0;
 		}
 
+		&--selected {
+			border: 2px solid $green;
+		}
+
+		&-selected-label {
+			color: $white !important;
+			margin: 0;
+			text-align: center;
+			padding: 5px;
+			text-transform: uppercase;
+			line-height: normal !important;
```

Comment:
```
.account-app p {
    font-size: 16px;
    line-height: 1.8;
    color: #3B3B3B;
}
```

Because this style is messing with it. We should get rid off it ASAP.